fission/src/mirabuf/MirabufParser.ts

```diff
@@ -393,7 +381,7 @@ class MirabufParser {
         let nextValue = 0
         const partTreeValues = new Map<string, number>()
 
-        const recursive = (partNode: mirabuf.INode) => {
+        function recursive(partNode: mirabuf.INode) {
```

Comment:
What was the reasoning behind this change? I was thinking that whenever we declare a nested function we should make it an arrow.

https://github.com/Autodesk/synthesis/blob/e8d823ba6918cd2b5a6c7cbc6de439bd5b96f7e3/fission/src/systems/physics/PhysicsSystem.ts#L346-L354

Reply:
I try to use it to indicate whether a function captures it's environment or whether it could be independent. I suppose we don't use it consistently at all, so i'm fine with sticking to arrow functions

Reply:
Totally makes sense, I'm fine with it either way.